Portugues.json

```
@@ -0,0 +1,152 @@
+{
+    "status": {
+        "terminated": "\nAplicacao Finalizada.\nFeche a janela para sair do programa.",
+        "goes_online": "{channel} esta ONLINE, Mudando...",
+        "goes_offline": "{channel} ficou OFFLINE, Mudando...",
+        "claimed_drop": "Coletou o drop: {drop}",
+        "claimed_points": "Coletou o bonus: {points}",
+        "earned_points": "Ganhou este valor de pontos: {points}, total: {balance}",
+        "no_channel": "Sem canal disponivel para assisitir. Esperando por um canal ONLINE...",
+        "no_campaign": "Sem campanhas ativas no momento para minerar. Esperando por uma campanha ativa..."
+    },
+    "login": {
+        "unexpected_content": "Algum erro inexperado aconteceu, geralmente por rediricionamento de internet. Voce precissa logar em algum site de internet para ter acesso?",
+        "incorrect_login_pass": "Senha ou usuario incorreto",
+        "incorrect_email_code": "Codigo de email incorreto",
+        "incorrect_twofa_code": "Codigo F2A incorreto",
+        "email_code_required": "Codigo no email requerido. Checar sua caixa de entrada",
+        "twofa_code_required": "Token F2A Requerido."
+    },
+    "error": {
+        "captcha": "Seu login foi negado pelo CAPTCHA.\nPor favor tente em aproximadamente +12 houras.",
+        "site_down": "Twitch esta fora do ar, tentando em {seconds} segundos...",
+        "no_connection": "Sem conecao com a TWITCH, reconectando em {seconds} segundos..."
+    },
+    "gui": {
+        "output": "Saida",
+        "status": {
+            "name": "Status",
+            "idle": "Em espera",
+            "exiting": "Saindo",
+            "terminated": "Finalizado",
+            "watching": "Assistindo o {channel}",
+            "cleanup": "Limpando os Canais...",
+            "gathering": "Coletando Canais...",
+            "switching": "Mudando o canal...",
+            "fetching_inventory": "Coletando Inventario...",
+            "fetching_campaigns": "Coletando campanhas... {counter}",

```

Per https://github.com/DevilXD/TwitchDropsMiner/commit/d308fad51cc3ff6d3f6c150692b102e68b4812a6

```suggestion
            "fetching_campaigns": "Coletando campanhas...",
```